src/main/java/com/github/phantomthief/util/MoreFunctions.java

```diff
@@ -22,10 +22,24 @@
         return catching(callable, e -> logger.error("", e));
     }
 
+    public static <X extends Exception> void catching(ThrowableRunnable<X> callable) {
```

Comment:
just reuse catching(Callable), callable only throws exception, so up bound is Exception